pallets/subtensor/src/coinbase/root.rs

```diff
@@ -908,7 +1002,7 @@ impl<T: Config> Pallet<T> {
     /// * 'NotEnoughBalanceToStake': If there isn't enough balance to stake for network registration.
     /// * 'BalanceWithdrawalError': If an error occurs during balance withdrawal for network registration.
     ///
-    pub fn user_add_network(
+    pub fn user_add_network_with_identity(
```

Comment:
most of code in the same with user_add_network. we can remove user_add_network, call user_add_network_with_identity with identity as None